Gemfile.lock

```diff
@@ -71,7 +71,7 @@ GEM
       safe_yaml (>= 1.0)
       sexp_processor (~> 4.7)
     builder (3.2.3)
-    byebug (10.0.1)
+    byebug (10.0.2)
```

Comment:
why have you changed it by hands?

Reply:
There was a conflict and i deseded to delete an older version